src/client/components/localisation/skybox/model.ts

```diff
@@ -0,0 +1,48 @@
+import { observable } from 'mobx'
+
+export class SkyboxModel {
+
+  @observable public turbidity: number
+  @observable public rayleigh: number
+  @observable public mieCoefficient: number
+  @observable public mieDirectionalG: number // default 0.8
+  @observable public luminance: number
+  @observable public inclination: number // elevation / inclination
+  @observable public azimuth: number // Facing front,
+  @observable public sun: boolean
+
+  public constructor(opts: SkyboxModelOpts) {
```

Comment:
You can delete the `SkyboxModelOpts` interface and use itself as a type.

e.g. `public constructor(opts: SkyboxModel) {`

(I only use this convenient shorthand for pure data models such as this, though)